Scripts/UpdateMFSGlobals.py

```diff
@@ -0,0 +1,210 @@
+# Update the globals that are updated by the Master File Server
+# in a running instance of VistA.
+#
+# The list of globals to update was taken from
+# http://www.vistapedia.com/index.php/MFS_list
+#
+# For detailed help information, please run
+#   python UpdateMFSGlobals.py -h
+#
+#---------------------------------------------------------------------------
+# Copyright 2016 The Open Source Electronic Health Record Agent
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#---------------------------------------------------------------------------
+
+from __future__ import with_statement
+import sys
+import os
+import re
+import glob
+import argparse
+import urllib
+from datetime import datetime
+import time
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+DEFAULT_CACHE_DIR = os.path.normpath(os.path.join(SCRIPT_DIR, "../"))
+sys.path.append(SCRIPT_DIR)
+
+
+from VistATestClient import VistATestClientFactory, createTestClientArgParser
+from LoggerManager import logger, initConsoleLogging, initFileLogging
+from VistATaskmanUtil import VistATaskmanUtil
+from VistAGlobalImport import VistAGlobalImport
+from ExternalDownloader import ExternalDataDownloader
+from PackRO             import pack
+from VistARoutineImport import VistARoutineImport
+
+  #(fileNum, Package, GlobalName)
+class updateMFSGlobals():
+  MFS_list = [
```

Comment:
Where did this list come from? Is it the one I provided from the STS group?

Reply:
As it used to mention, the first list came from VistApedia: http://vistapedia.net/index.php/MFS_list

Since that didn't have any information about the IMMUNIZATION file, I figured it was out of date and looked into VistA.  I saw that the entries in Master File Parameters had all the ones in my list plus some extra, so I used that as the new list:
> Select OPTION: 5  INQUIRE TO FILE ENTRIES
>  
>  
>  
> Output from what File: MASTER FILE PARAMETERS//   (40 entries)
> Select MASTER FILE PARAMETERS: ??
>  
>    Choose from:
>    MASTER FILE PARAMETERS
>    MD5 Signature
>    STANDARD TERMINOLOGY VERSION FILE
>    RACE MASTER
>    MASTER MARITAL STATUS
>    MASTER RELIGION
>    DRUG INGREDIENTS
>    VA GENERIC
>    VA DRUG CLASS
>    MASTER DOSAGE FORM
>    VA PRODUCT
>    STANDARD MEDICATION ROUTES
>    MASTER LABORATORY TEST
>    MASTER RADIOLOGY PROCEDURE
>    ORDER STATUS
>    NATURE OF ORDER
>    GMRV VITAL TYPE
>    GMRV VITAL QUALIFIER
>    GMRV VITAL CATEGORY
>    GMR ALLERGIES
>    SIGN/SYMPTOMS
>    MASTER TYPE OF PLAN
>    MAPPINGS
>    VACCINE INFORMATION STATEMENT
>    IMMUNIZATION INFO SOURCE
>    IMM ADMINISTRATION ROUTE
>    IMM ADMINISTRATION SITE (BODY)
>    IMM CONTRAINDICATION REASONS
>    IMM REFUSAL REASONS
>    TIU STATUS
>    TIU VHA ENTERPRISE STANDARD TITLE
>    TIU LOINC SUBJECT MATTER DOMAIN
>    TIU LOINC ROLE
>    TIU LOINC SETTING
>    TIU LOINC SERVICE
>    TIU LOINC DOCUMENT TYPE
>    PERSON CLASS
>    IMM MANUFACTURER
>    IMMUNIZATION
>    SKIN TEST